src/api/rpc_api/state.rs

```diff
@@ -73,36 +87,51 @@ pub trait GetStorage<Hash> {
 		at_block: Option<Hash>,
 	) -> Result<Vec<StorageKey>>;
 
-	fn get_opaque_storage_by_key_hash(
+	/// Retrieve the raw storage for the given `storage_key`.
+	///
+	/// `at_block`: the state is queried at this block, set to `None` to get the state from the latest known block.
+	fn get_opaque_storage(
```

Comment:
Above, the `_by_key` is used whenever a storage key is required as an input. If there's no `by_key`, then the pallet and storage item are required. Is this convention break here intentional?

Reply:
Thanks for this observation. Will change it.